src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Entities/TournamentEntity.cs

```diff
@@ -81,10 +82,10 @@ public TournamentEntity()
         /// <summary>
         /// Gets or sets collection of tournaments divisions
         /// </summary>
-        public virtual List<DivisionEntity> Divisions
+        public virtual ICollection<DivisionEntity> Divisions
         {
             get => _divisions;
-            set => _divisions = value;
+            set => _divisions = value.ToList();
```

Comment:
?

Reply:
ok, one moment

Reply:
only way is to revert from ICollection<T> to List<T>